lisa/sut_orchestrator/azure/tools.py

```diff
@@ -108,8 +108,8 @@ def is_swap_enabled(self) -> bool:
                 f"Unknown value for ResourceDisk.EnableSwap : {is_swap_enabled}"
             )
 
-    def is_rdma_enabled(self) -> bool:
-        waagent_configuration = self.get_configuration()
+    def is_rdma_enabled(self, force_run: bool = False) -> bool:
```

Comment:
Because all caller of `is_rdma_enabled` need `force_run=True`. It doesn't need to expose force_run here, just set it to True when calling get_configuration.

Reply:
updated